code/game/machinery/doors/airlock.dm

```diff
@@ -851,10 +851,16 @@ About the new airlock wires panel:
 /obj/machinery/door/airlock/proc/toggle_bolt(mob/user)
 	if(wires.is_cut(WIRE_DOOR_BOLTS))
 		to_chat(user, "<span class='warning'>The door bolt control wire has been cut - Door bolts permanently dropped.</span>")
-	else if(lock())
-		to_chat(user, "<span class='notice'>The door bolts have been dropped.</span>")
-	else if(unlock())
+		return
+
+	if(unlock()) // Trying to unbolt
 		to_chat(user, "<span class='notice'>The door bolts have been raised.</span>")
+		return
+
+	if(lock()) // Trying to bolt
+		to_chat(user, "<span class='notice'>The door bolts have been dropped.</span>")
+		user.create_log(MISC_LOG, "Bolted", src)
+		add_hiddenprint(user)
```

Comment:
```suggestion
		add_hiddenprint(user)
		return
		
	to_chat(user, "<span class='warning'>No action was performed. This may be a bug.</span>")
```

Reply:
It seems like trying to unbolt the door while the power is disabled will also trigger this, so it might be best to just leave it out.